src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGatewaySkuTier.cs

```diff
@@ -21,6 +21,9 @@ public static class VirtualNetworkGatewaySkuTier
         public const string HighPerformance = "HighPerformance";
         public const string Standard = "Standard";
         public const string UltraPerformance = "UltraPerformance";
+        public const string VpnGw1 = "VpnGw1";
```

Comment:
@ritwikbasu can you link the swagger spec for this change?
Also what is still not clear is, how come the sdk branch as 9.1.0-preview as the latest version of network package, but what is published out on nuget is 10.0.0-preview?